### DISCUSSION TITLE
not able to push changes- error: cannot publish unborn Head on desktop

### DISCUSSION THREAD

**Discussion by gsahab0007** [community/community#65410] [Repositories] [answered]
### Select Topic Area

Question

### Body

while push changes on desktop -- showing - current branch: (unknown) - publish Branch: cannot publish unborn Head

**Comment by seyLu** [comment, marked as answer]
- your repo is empty
- try to create at least a README.md file, then `add > commit > push` that change


stackoverflow ref:
https://stackoverflow.com/questions/48527025/github-website-publish-cannot-publish-unborn-head

**Reply by gsahab0007** [reply]
I got solution - i created new repository and put all previous code in that -- it worked
thanks for response.

**Comment by gsahab0007** [comment]
i got solution as above so closing query. thanks

**Reply by seyLu** [reply]
Cool. Glad I could help.
